src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -23,7 +23,7 @@ internal static bool IsSupportedTarget(EnvironmentVariableTarget target)
                 return false;
             }
 
-            return target == EnvironmentVariableTarget.Process || (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && !PlatformDetection.IsUap);
+            return true;
```

Comment:
Are you sure this is right?  We don't support User/Machine on unix.

Reply:
This is the reason for my test failures. Thanks.